src/backend/base/langflow/graph/utils.py

```diff
@@ -124,12 +125,26 @@ async def log_transaction(
             else:
                 return
         inputs = _vertex_to_primitive_dict(source)
+
+        # Convert the result to a serializable format
+        if source.result:
+            try:
+                result_dict = json.loads(source.result.model_dump_json())
```

Comment:
I think you can just use `model_dump` and make sure the serializer for this type calls the `serialize` function.